sbom/examples/rpm/release/add_release_data.py

```diff
@@ -3,6 +3,21 @@
 
 from packageurl import PackageURL
 
+# With help from https://security.access.redhat.com/data/meta/v1/repository-to-cpe.json
+cpe_map = {
+    "openshift-pipelines-client-1.14.3-11352.el8": ["cpe:/a:redhat:openshift_pipelines:1.15::el8"],
+    "openssl-3.0.7-18.el9_2": [
+        "cpe:/a:redhat:rhel_eus:9.2::appstream",
+        "cpe:/o:redhat:rhel_eus:9.2::baseos" "cpe:/o:redhat:rhel_e4s:9.2::baseos",
```

Comment:
Missing a comma here.

Reply:
This has now been replaced with a list of SPDX packages filled in with variant information from Errata Tool.